### PR TITLE
Added new line including circle-map and biopython; Circle-Map needs biopython=1.77 to work

### PR DESCRIPTION
**Pull request by DSchreyer**

### CODE REVIEW

**Comment by bgruening** [comment]
Should that not be fixed in the conda package?

**Comment by DSchreyer** [comment]
Technically yes, but the software is not updated, yet, to fix this issue.

**Comment by DSchreyer** [comment]
What do I need to do to get this pull request accepted? The current circle-map version is not working with biopython=1.78 and therefore needs 1.77. The Circle-Map author published version 1.1.5 onto github last year, but it was not updated on conda. Therefore, a mulled container would be good to have until everything is resolved.

**Comment by DSchreyer** [comment]
Mulled build and test outputs the following mulled container hash:
mulled-v2-3423532b7f92d89bbf04791d2e48d40cb0a4111e:2a8abad654fa8c270bfcc01eef47487e2573badb-0

File: combinations/hash.tsv [hunk 1]
```diff
@@ -202,3 +202,5 @@ biscuit=1.0.2.20220113,samtools=1.15,samblaster=0.1.26,parallel=20211222
 samtools=1.14.0,bedtools=2.30.0,ucsc-facount=377,python=3.8
 gatk4=4.2.0.0,r-base=4.1,r-ggplot2=3.3.5,datamash=1.1.0
 pyliftover=0.4,pandas=1.1.5
+python=2.7,pysam=0.17.0,flask=1.1.2,cython=0.29.14,numpy=1.16.6,scipy=1.2.1,matplotlib=2.2.5,mosek=8.0.60
```
Comment (bgruening) [inline combinations/hash.tsv:205]: is that needed? this does not look like a tool based container

Comment (DSchreyer) [reply combinations/hash.tsv]: No, I removed it.

**Comment by bgruening** [comment]
Thanks @DSchreyer

**Pull request merged**